fastexport/jit.py

```diff
@@ -8,24 +8,20 @@
 from fastai.learner import *
 from fastai.torch_core import TensorBase
 
-# Internal Cell
-@patch
-def requires_grad_(self:TensorBase, requires_grad=True):
-    # Workaround https://github.com/pytorch/pytorch/issues/50219
-    self.requires_grad = requires_grad
-    return self
-
 # Cell
 mk_class('JitMode', **{o:o.lower() for o in ['Trace','Script']},
          doc="All possible export modes as attributes to get tab-completion and typo-proofing")
 
 # Cell
 @patch
-def to_jit(self:Learner, fname='export.pt', mode=JitMode.Trace):
+def to_jit(self:Learner, fname='torchscript_model.ts', mode=JitMode.Trace, cpu=False):
```

Comment:
Let's get rid of `cpu=False`. `learn.export` will handle the device, so switch that to `device` and we can default it to `cpu`